model/modelpb/span.pb.json.go

```diff
@@ -0,0 +1,105 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package modelpb
+
+import (
+	"time"
+
+	"github.com/elastic/apm-data/model/internal/modeljson"
+)
+
+func (db *DB) toModelJSON(out *modeljson.DB) {
+	*out = modeljson.DB{
+		Instance:     db.Instance,
+		Statement:    db.Statement,
+		Type:         db.Type,
+		Link:         db.Link,
+		RowsAffected: db.RowsAffected,
+		User:         modeljson.DBUser{Name: db.UserName},
+	}
+}
+
+func (c *Composite) toModelJSON(out *modeljson.SpanComposite) {
+	sumDuration := time.Duration(c.Sum * float64(time.Millisecond))
+	*out = modeljson.SpanComposite{
+		CompressionStrategy: c.CompressionStrategy.String(),
```

Comment:
The value should be "exact_match" or "same_kind", I don't think this is right

Reply:
Fixed!